.github/workflows/ci.yml

```diff
@@ -608,6 +608,24 @@ jobs:
           testing/bin/ptl suite run \
             --suite ${{ matrix.suite }} --config config-${{ matrix.config }} --bind=off --logs-dir logs/ --timeout 2h \
             --trino-jdk-version zulu_${{ matrix.jdk }}
+      - name: Generate Jacoco report
+        run: |
+          export TRINO_VERSION=$(./mvnw help:evaluate -Dexpression=project.version -q -DforceStdout)
+          mkdir -p /tmp/trino-jacoco/reports
+          mkdir -p "core/trino-server/target/trino-server-${TRINO_VERSION}-trino-jars"
+          # copy only trino JARs, since we don't care about coverage of external dependencies
+          find "core/trino-server/target/trino-server-${TRINO_VERSION}-hardlinks" -name 'trino-*.jar' -print0 | xargs --null cp --link --no-clobber -t "core/trino-server/target/trino-server-${TRINO_VERSION}-trino-jars"
+          java -jar testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/bin/jacococli.jar \
```

Comment:
It's not a great idea to check in these jacoco libraries into the repository. It makes it harder to track what version we're using, when to upgrade (these libraries are notably fragile with respect to JVM bytecode spec changes, for instance).

Can we get make them proper maven dependencies?